docs/reference/api/pieces.md

```diff
@@ -4,23 +4,23 @@ Every [piece type](TODO) has built in REST end points that share their overall s
 
 ## Endpoints
 
-**Note:** `:piece-name` represents the `name` option for a piece type. For example, you would request all pieces of an `article` piece type with `/api/v1/article`.
+**Note:** `:piece-type` represents the `name` option for a piece type. For example, you would request all pieces of an `article` piece type with `/api/v1/article`.
```

Comment:
Really think it is worth saying `piece-module-name` instead here, this is what they need to know in order to be 100% sure of getting this URL right on the first try and avoiding frustration.

Reply:
If `name` is important in the placeholder, then `:piece-name` should work. We could make the description more explicit, with "`:piece-name` represents the name of a piece-type module." I'm trying to avoid the extra verbosity and avoid beating people over the head with "module." This is something we've discussed a few times and documented in our internal docs outline:
> Acknowledge that everything is a module and what the types are. After that, don't worry about saying "module" all the time (e.g., "create a widget" vs "create a widget module"). Don't make it sound more complex than it is.